src/query.ts

```diff
@@ -15,6 +15,9 @@ import { download } from "./download";
 interface Repo {
   id: number;
   nwo: string;
+  downloadUrl?: string;
+
+  // token and pat are deprecated
   token?: string; // SignedAuthToken
   pat?: string;
```

Comment:
`pat` is currently used (only) by the integration tests to download databases specified by the `instructions.json` contained in a release on this repository: https://github.com/github/codeql-variant-analysis-action/releases/tag/test

```json
{
    "repositories": [
        {
            "id": 376068344,
            "nwo": "dsp-testing/qc-demo-github-certstore"
        },
        {
            "id": 211169016,
            "nwo": "github/vscode-codeql"
        }

    ],
    "repoNwoChunks": [
        [
            "dsp-testing/qc-demo-github-certstore",
            "github/vscode-codeql"
        ]
    ]
}
```

One repository is public, one is private and will always require the bot PAT. Or we could definitely discuss other ways of integration testing!

Reply:
Good point we'll have to keep the path for calling the `GET .../databases/:language` endpoint to support the integration tests. That's a little sad that the tests will be using a slightly different code path that the production case, but it will be limited to a `if` block of only a few lines, so maybe it'll be acceptable, and it's still a HTTP request but just to a different place. 🤔 

For now I'm not worried, but we'll just have to keep the code for handling the PAT around when we clean this area up later.